packages/wix-style-react/src/SelectableAccordion/Item/Item.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import { WixStyleReactContext } from '../../WixStyleReactProvider/context';
 import Checkbox from '../../Checkbox';
 import ToggleSwitch from '../../ToggleSwitch';
-import RadioGroup from '../../RadioGroup';
+import Radio from '../../Radio/Radio';
```

Comment:
A component should be imported from the index
```suggestion
import Radio from '../../Radio';
```